whipper/command/mblookup.py

```diff
@@ -12,35 +14,56 @@ class MBLookup(BaseCommand):
 
     def add_arguments(self):
         self.parser.add_argument(
-            'mbdiscid', action='store', help="MB disc id to look up"
+            'id', action='store', help="MB disc id or release id to look up"
         )
 
+    def _printMetadata(self, md):
+        """
+        Print out metadata received in a sensible way
+
+        :param md: MusicBrainz's metadata about the disc
+        :type md: `DiscMetadata`
+        rtype: None
+        """
+        print('    Artist: %s' % md.artist.encode('utf-8'))
+        print('    Title:  %s' % md.title.encode('utf-8'))
+        print('    Type:   %s' % str(md.releaseType).encode('utf-8'))
+        print('    URL:    %s' % md.url)
+        print('    Tracks: %d' % len(md.tracks))
+        if md.catalogNumber:
+            print('    Cat no: %s' % md.catalogNumber)
+        if md.barcode:
+            print('    Barcode: %s' % md.barcode)
+
+            for j, track in enumerate(md.tracks):
+                print('      Track %2d: %s - %s' % (
+                    j + 1, track.artist.encode('utf-8'),
+                    track.title.encode('utf-8')
+                ))
+        return None
+
     def do(self):
         try:
-            discId = str(self.options.mbdiscid)
+            id = str(self.options.id)
```

Comment:
May not be fully in scope for this pull request but it may be a good idea to use the [`strip()`](https://docs.python.org/3.5/library/stdtypes.html#str.strip) method in this line:

```python
id = str(self.options.id).strip()
```

Reply:
Yes i have added it anyway :D.